src/gui/transferlistwidget.cpp

```diff
@@ -1142,11 +1153,15 @@ void TransferListWidget::displayListMenu()
     if (addedPreviewAction)
         listMenu->addSeparator();
     if (oneHasMetadata)
-    {
         listMenu->addAction(actionForceRecheck);
-        listMenu->addAction(actionForceReannounce);
-        listMenu->addSeparator();
-    }
+    // We can not force reannounce torrents that are paused/errored/checking/missing files/queued.
+    // We may already have the tracker list from magnet url without metadata.
+    // So we can force reannounce anyway.
```

Comment:
The comments above confuse me. `We can not force reannounce... So we can force reannounce anyway.`

Reply:
You probably meant that we can technically reannounce the mentioned torrents, but we should prevent the user from doing this, since it is meaningless/incorrect/etc., right?
BTW, what happens if a user chooses several torrents, some of which we can reannounce and others not?

Reply:
Currently if we don't have metadata we can't force reannounce.
But with this change we can force reannounce even without metadata.